specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,716 @@
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Register GeoJSON data containing geometries that represent a collection of geofences to the Azure Maps Service": {
+            "$ref": "./examples/DataRegistries_Register.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/OperationIdHeader"
+          },
+          {
+            "name": "dataRegistry",
+            "in": "body",
+            "description": "The data registry to be registered.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataRegistry"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Resource Creating: The request has been accepted for processing. Please use the URL in the Operation-Location Header to obtain status.",
+            "headers": {
+              "Operation-Id": {
+                "type": "string",
+                "description": "ID of the status monitor for the operation."
+              },
+              "Operation-Location": {
+                "type": "string",
+                "description": "New URL to check for the results of the long running process. `api-version` is not included in the URL."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Metadata"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      },
+      "get": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of DataRegistry APIs is bound to the region of your maps account. Global maps account is not be able to use this service.\n\nThis API allows the caller to fetch a list of all data registries registered previously using the [DataRegistry Register API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register).\n\nThis API won't check whether Azure Maps can access the underlying data and whether integrity check is passed for each data registry. Please use [DataRegistry GetMetadata API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/get-metadata) for this purpose.\n\n## Submit List Request\n\nTo list all your map data content you will issue a `GET` request with no additional parameters.\n\n## List Response\n\nThe Data List API returns the complete list of all data in `json` format.",
```

Comment:
> The Data List API returns the complete list of all data in json format.

The `Data` List here may be DataRegistry List API.

Reply:
Fixed